synapse/storage/databases/main/end_to_end_keys.py

```diff
@@ -930,14 +990,26 @@ def _claim_e2e_one_time_key_returning(
 
 
 class EndToEndKeyStore(EndToEndKeyWorkerStore, SQLBaseStore):
+    def __init__(
+        self,
+        database: DatabasePool,
+        db_conn: LoggingDatabaseConnection,
+        hs: "HomeServer",
+    ):
+        super().__init__(database, db_conn, hs)
+
+        self._cross_signing_id_gen = StreamIdGenerator(
+            db_conn, "e2e_cross_signing_keys", "stream_id"
+        )
```

Comment:
> Moved the `_cross_signing_id_gen` definition here, since it's only used by this class.

This gives me a very strong, warm, fuzzy feeling inside. Thank you!